src/services/wallet-client/WalletClient.ts

```diff
@@ -26,6 +26,21 @@ class WalletClient {
     }
   }
 
+  async getCurrentWAudioBalance(): Promise<BNWei> {
+    try {
+      const balance = await AudiusBackend.getWAudioBalance()
+      return balance as BNWei
+    } catch (err) {
+      console.log(err)
```

Comment:
console log right?

Reply:
Yeah, can change to error - will also change surrounding functions function.